galaxy/Dockerfile

```diff
@@ -199,4 +203,4 @@ EXPOSE :9002
 VOLUME ["/export/", "/data/", "/var/lib/docker"]
 
 # Autostart script that is invoked during container start
-CMD ["/usr/bin/startup"]
+CMD ["/usr/bin/startCondor.sh"]
```

Comment:
I bet if I instead `source /usr/bin/startup` then the magic will still work. I'll do that real quick.